lib/fizzbuzz.rb

```diff
@@ -1,9 +1,13 @@
 class SuperFizzBuzz
 
   def run(input)
+    raise ArgumentError.new("Argument is not an Integer") unless input.instance_of? Fixnum
```

Comment:
You might want to change this to `Numeric` in case someone uses a different kind of number (but double check that `Numerics` still have the methods you need).